storage/internal/benchmarks/download_benchmark.go

```diff
@@ -25,12 +25,13 @@ import (
 )
 
 type downloadOpts struct {
-	client      *storage.Client
-	objectSize  int64
-	bucket      string
-	object      string
-	rangeStart  int64
-	rangeLength int64
+	client              *storage.Client
+	objectSize          int64
+	bucket              string
+	object              string
+	rangeStart          int64
+	rangeLength         int64
+	downloadToDirectory string
```

Comment:
Can you explain how adding this opt is related? Is it something that's part of the public surface or just intended for testing?

Reply:
Sounds good, maybe just note this in the commit msg as well.